third_party/protobuf/3.6.1/BUILD

```diff
@@ -24,9 +27,6 @@ config_setting(
 load(":protobuf.bzl", "py_proto_library")
 load(":compiler_config_setting.bzl", "create_compiler_config_setting")
 
-# Needed for #9006. Hopefully a future upstream version will include this line.
```

Comment:
I sent a PR to Protobuf to fix this, so I don't think this comment is needed.

Reply:
Wow. You're awesome :)